docs/docusaurus/versioned_docs/version-1.0.0/orc8r/deploy_install.md

```diff
@@ -297,36 +297,25 @@ metrics:
       worker-type: metrics
```

Comment:
We shouldn't update this doc as the changes won't exist for the version-1.0.0 tag

Reply:
fixed